libres/lib/include/ert/analysis/update.hpp

```diff
@@ -17,5 +17,15 @@ bool smoother_update(std::vector<int> step_list,
                      ensemble_config_type *ensemble_config,
                      enkf_fs_type *source_fs, enkf_fs_type *target_fs,
                      FILE *log_stream, bool verbose);
+void save_parameters(enkf_fs_type *target_fs,
+                     ensemble_config_type *ensemble_config,
+                     const int_vector_type *iens_active_index, int last_step,
+                     const local_ministep_type *ministep, matrix_type *A);
```

Comment:
Could the input-matrix `A` be defined as `const`?

Reply:
since it is passed to the serialize_info_alloc wich is also used in load_parameters it can not be const

Reply:
Is this something we should consider redesigning as it is a bit hard to parse, for me at least.
I would expect a function called `save_parameters` to be able to take `const matrix_type *A`.
Not saying we should do it in this PR!